### ISSUE TITLE
Unable to open repository

### ISSUE THREAD

**Issue by flip111** [hartwork/svneverever#8]
I tried the following commands

```bash
svn co https://plugins.svn.wordpress.org/schreikasten
svneverever schreikasten
```

i get
```
ERROR: Unable to connect to a repository at URL 'file:///home/flip111/php/tmp/schreikasten'
Unable to open repository 'file:///home/flip111/php/tmp/schreikasten'
```

What are the usage instructions? Should i follow the example literally and create a dummy repository?

**Comment by hartwork** [comment]
Hi @flip111 ,

the problem you're running into is that `svn checkout` gives you a checkout with the latest commit but not full history whereas svneverever needs the "server side" of the repository with full history. If you don't have access to the server you can use rsvndump to re-create a repository "backend" locally. The readme has [an example](https://github.com/hartwork/svneverever#example) on that topic.

Best, Sebastian

**Comment by flip111** [comment]
Thank you @hartwork i get the output as in the example now. Maybe it would be a good idea to mention in the readme that `rsvndump` is a dependency and the command that is used in the example is the normal way to use `svneverever`.

**Comment by hartwork** [comment]
You could have svneverever access https://plugins.svn.wordpress.org/schreikasten directly, it will just be too slow and waste resources for more than a one-time execution. rsvndump makes things go faster primarily, it's not a dependency, i.e. svneverever functions and runs fine without it.

**Comment by flip111** [comment]
Can i open a PR to reword the readme to make the two available options more explicit ?

When svneverever accesses the server directly will it put more load on the server than rsvndump ?

**Comment by hartwork** [comment]
> Can i open a PR to reword the readme to make the two available options more explicit ?

Sure, but expect picky review :+1: 


> When svneverever accesses the server directly will it put more load on the server than rsvndump ?

When use svneverever to ask questions to the history of a remote repository you can choose:
- a) use of rsvndump downloads history once, then works offline
- b) not using rsvndump downloads full history for every call.

So `O(1)` versus `O(n)` network traffic.

**Comment by hartwork** [comment]
@flip111 I have further improved reporting on this mix-up in commit a1feb0d9ec4d4a75bdf6aa0d2b6ed2979e15c81d now and released version 1.7.0 including this improvement up here and on PyPI. I'm closing the ticket as fixed.
